### PR TITLE
[Audit Issues A&B] Validate ValidatorAddedEvent operators

### PR DESCRIPTION
**Pull request by nkryuchkov**
Add validation for `ValidatorAddedEvent`, fail if:
- too many operators (>13)
- 0 operators
- not 3f+1
- duplicates
- doesn't exist in DB

### CODE REVIEW

File: operator/validator/event_handler.go
Comment (nkryuchkov) [reply operator/validator/event_handler.go:146]: @y0sher yes, we need to discuss what validations we want to do. They may be included in this PR or go in a new PR

**Comment by y0sher** [comment]
This PR kinda interacts with #975 , we need to see how we want to add them combined or separately. @olegshmuelov @lior-blox .
on hold for now

**Comment by y0sher** [comment]
> This PR kinda interacts with #975 , we need to see how we want to add them combined or separately. @olegshmuelov @lior-blox . on hold for now

@nkryuchkov , #974 was merged finally. please base this PR on top of updated code and make sure the changes are aligned. after then I'll approve and we'll merge.

File: operator/validator/event_handler.go
Comment (nkryuchkov) [inline operator/validator/event_handler.go:503]: moved to operator/validator/event_validation.go

**Review by y0sher — DISMISSED** [review]

**Comment by moshe-blox** [comment]
TODO: check that event's operators actually exist

**Comment by nkryuchkov** [comment]
> TODO: check that event's operators actually exist

@moshe-blox done

File: eth/eventhandler/validation.go
Comment (moshe-blox) [inline eth/eventhandler/validation.go:33]: Can we please move this func under [this one](https://github.com/bloxapp/ssv/blob/90874aefde6f266d476f6c74ff3ed066ed136b58/protocol/v2/types/ssvshare.go#L82-L85) (maybe call it `ValidCommitteeSize`) and add a test for it there?

Reason is because it's related to that func so it's better together with it (so if we make modification to any of them we can't miss the other).

**Review by moshe-blox — CHANGES_REQUESTED** [review]
really good test, gj 💪

File: eth/eventhandler/validation.go
Comment (nkryuchkov) [reply eth/eventhandler/validation.go]: done

**Review by moshe-blox — APPROVED** [review]
👍

**Review by moshe-blox — APPROVED** [review]

**Pull request merged**
